CONTRIBUTING.md

```diff
@@ -1,35 +1,225 @@
+
+*   Specify **how you installed**:
+    -   Linux: either from the PPA, your distribution's repositories, or compiled from source, or even possibly third-party repositories.
+    -   Windows: either from the installer, or compiled from source, or even possibly third-party repositories.
+    -   macOS: either from the installer, or compiled from source, or even possibly third-party repositories.
+
+
+*   **Describe the exact steps which reproduce the problem in as many details as possible.**
+    -   For example, start by explaining how you started qBittorrent, e.g. was it via the terminal? Desktop icon? Did you start it as root or normal user?
+    -   **When listing steps, don't just say what you did, but explain how you did it.**
+        -   For example, if you added a torrent for download, did you do so via a `.torrent` file or via a magnet link? If it was with a torrent file did you do so by dragging the torrent file from the file manager to the transfer list, or did you use the "Add Torrent File" in the Top Bar?
+    -   Describe the behavior you observed after following the steps and point out what exactly is the problem with that behavior; this is what we'll be looking for after executing the steps.
+
+
+*   **Explain which behavior you expected to see instead** and why.
+
+*   Use **screenshots/animated GIFs to help describe the issue** whenever appropriate [(How?)][attachments-howto-url].
+
+*   If the problem wasn't triggered by a specific action, describe what you were doing before the problem happened.
+
+*   **If you are reporting that qBittorrent crashes**, include the stack trace in the report; include it in a code block, a file attachment, or put it in a gist and provide link to that gist.
+
+*   **For performance-related issues**, include as much profiling data as you can (resource usage graphs, etc).
+
+*   Paste the **qBittorrent log** (or put the contents of the log in a gist and provide a link to the gist). The log can be viewed in the GUI (View -> Log -> tick all boxes). If you can't do that, the file is at:
+    -   Linux: `~/.local/share/data/qBittorrent/logs/qBittorrent.log`
+    -   Windows: `%LocalAppData%\qBittorrent\logs`
+    -   macOS: `~/Library/Application Support/qBittorrent/qBittorrent.log`
+
+
+*   **Do NOT post comments like "+1" or "me too!"** without providing new relevant info on the issue. Using the built-in reactions is OK though. Remember that you can use the "subscribe" button to receive notifications of that report without having to comment first.
+
+*   If there seems to be an **issue with specific torrent files/magnet links**:
+    -   Don't post or private `.torrent` files/magnet links, or ones that point to copyrighted content. If you are willing, offer to email a link or the `.torrent` file itself to whoever developer is debugging it and requests it.
```

Comment:
Don't post _public_ or private....?

Reply:
@thalieht 
That first "or" shouldn't actually be there. Fixed